src/graphviz-1-remove-decl.patch

```diff
@@ -0,0 +1,398 @@
+diff -ur a/ast_common.h b/ast_common.h
+--- a/ast_common.h	2016-08-11 20:45:52.985309265 +1000
++++ b/ast_common.h	2016-08-11 20:52:48.433612672 +1000
+@@ -134,10 +134,10 @@
+ /* dynamic linked library external scope handling */
+ #undef extern
+ #if _dll_import && !defined(__EXPORT__) && _DLL_BLD
+-#define __EXPORT__      __declspec(dllexport)
```

Comment:
All these `__declspec` changes should be wrapped in macros in a single top-level header and then used throughout. 

See https://github.com/mxe/mxe/blob/c92ea065fb3773f90cf91983835b7b47a335d591/src/libepoxy-1-fixes.patch#L20-L31 for an example.